### DISCUSSION TITLE
Game freezes and DC file size

### DISCUSSION THREAD

**Discussion by FrankMora** [vezel-dev/novadrop#25] [Questions] [answered]
Question about file size. Before packing, the file size is about 60 mb, after packing it about 120 mb. Is it because your packing algorithm has duplicate elements? As far as I know, links to elements are created in the original algorithm.

With the repackaged DC, all menus and opening items in the game freeze for a few seconds, while with the original DC, there is no such problem.

**Comment by FrankMora** [comment]
Video with this problem
Repacked (without any changes) DC: https://disk.yandex.ru/i/z9RsDalUAV7zxQ
Original DC: https://disk.yandex.ru/d/BlgrcGsNKr_rcw

**Comment by alexrp** [comment, marked as answer]
> Question about file size. Before packing, the file size is about 60 mb, after packing it about 120 mb. Is it because your packing algorithm has duplicate elements? As far as I know, links to elements are created in the original algorithm.

Yes. See #21.

> With the repackaged DC, all menus and opening items in the game freeze for a few seconds, while with the original DC, there is no such problem.

Investigating this in #26.

**Comment by alexrp** [comment]
e2110a70e6aa59d54121197c6e51868e0e346b9c should fix the freeze issue.
